src/main/java/com/stefancooper/SpigotUHC/resources/ConfigKey.java

```diff
@@ -1,7 +1,26 @@
 package com.stefancooper.SpigotUHC.resources;
 
 public enum ConfigKey {
+    //World Border Enums
     WORLD_BORDER_INITIAL_SIZE("world.border.initial.size"),
+    WORLD_BORDER_FINAL_SIZE("world.border.final.size"),
+    WORLD_BORDER_SHRINKING_PERIOD("world.border.shrinking.period"),
+    WORLD_BORDER_GRACE_PERIOD("world.border.grace.period"),
+    WORLD_BORDER_CENTER_X("world.border.center.x"),
+    WORLD_BORDER_CENTER_Y("world.border.center.y"),

```

Comment:
i also don't know if we need a world center y, cos you can't set that in the spigot api

Reply:
removed